libtenzir/builtins/connectors/tcp.cpp

```diff
@@ -131,24 +130,26 @@ auto make_tcp_bridge(tcp_bridge_actor::stateful_pointer<tcp_bridge_state> self)
             caf::anon_send(
               caf::actor_cast<caf::actor>(hdl),
               caf::make_action([self, ec, endpoint]() mutable {
-                if (ec)
+                if (ec) {
                   return self->state.connection_rp.deliver(caf::make_error(
                     ec::system_error,
                     fmt::format("connection failed: {}", ec.message())));
+                }
                 if (self->state.tls_socket) {
                   self->state.tls_socket->handshake(
                     boost::asio::ssl::stream<
                       boost::asio::ip::tcp::socket>::client,
                     ec);
+                  if (ec) {
+                    return self->state.connection_rp.deliver(caf::make_error(
```

Comment:
IIuc this `return` is the only non-cosmetic change, is that correct?

Reply:
Some `return`s are cosmetic transformations for consistency throughout a single function, others not. For example, a failed TLS handshake caused a double delivery.